app-frontend/src/app/pages/admin/platform/settings/email/email.html

```diff
@@ -0,0 +1,66 @@
+<div class="content column-10">
+  <div class="dashboard-header">
+    <h1 class="h3 page-title">Email Notifications</h1>
+  </div>
+  <form name="emailSettings">
+    <div class="form-group">
+      <label for="name">User Name</label>
+      <input id="name"
+             type="email"
+             class="form-control"
+             name="email"
+             ng-attr-placeholder="{{$ctrl.platformBuffer.publicSettings.emailUser.length ?
+               $ctrl.platformBuffer.publicSettings.emailUser : 'user@example.com'}}"
+             ng-model="$ctrl.platformBuffer.publicSettings.emailUser"
+             required>
+      <span class="error color-danger" ng-show="emailSettings.email.$error.required">required</span>
+      <span class="error color-danger" ng-show="emailSettings.email.$error.email">invalid email</span>
+    </div>
+    <div class="form-group">
+      <label for="password">Password</label>
```

Comment:
You can add `autocomplete="new-password"` to prevent the password field from getting autocompleted in chrome